pkg/api/response/response.go

```diff
@@ -292,6 +296,10 @@ func ErrOrFallback(status int, message string, err error) *NormalResponse {
 		return Err(err)
 	}
 
+	if errors.Is(err, context.Canceled) {
```

Comment:
what if a grafanaErr (for any reason) wraps a context.Canceled error? I guess that's possible and in that case this condition is not reachable.
are we ok with this?

Reply:
Good point. For now I guess yes since if there's a grafanaErr someone has made a specific decision to possibly return a different status code is my thinking.